File: Yarışma Cevapları/Abdulkadir Yıldırım - Hüseyin Özcan/soru1.py

def usAl(taban,us):

    if taban == "x" or us == "x":
        return 0
    else:
        k= int(taban)**int(us)
        print(k)
        return usAl((input("Taban:")),(input("üs: ")))

print(usAl(0,1))

File: Competitor Answers/Güney KAYA/Soru_1.cs
using System;

namespace Hackaton_2
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine(UsAl(2, 5));
        }
        static int UsAl(int taban, int us)
        {

            if (us == 0)
            {
                return 1;
            }
            else
            {
                return taban * UsAl(taban, us - 1);
            }
        }
    }
}
